portfolio.html

```diff
@@ -7,5 +7,60 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="logo">
+          <a href="#">My Logo</a>
+        </div>
+        <nav>
+          <div class="menu-icon">
+            <img src="images/Union.png" alt="menu button" />
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="headline">
+          <h2 class="title">I’m Yaseen Glad to see you!</h2>
```

Comment:
- [ ] [OPTIONAL] This header should be an `<h1>` kindly consider changing it so your page has one `<h1>` tag